### PR TITLE
remove new references on non-classes

### PR DESCRIPTION
**Pull request by ricardo-devis-agullo**
Removing `new` references on functions that return normal objects (not classes), since it's not needed.

### CODE REVIEW

File: src/cli/facade/dev.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ module.exports = function(dependencies) {
                 return callback(err);
               }
 
-              const registry = new oc.Registry({
+              const registry = oc.Registry({
```
Comment (pbazydlo) [inline src/cli/facade/dev.js:160]: won't this change break how registry is initialised right now outside of this repository?

Comment (ricardo-devis-agullo) [reply src/cli/facade/dev.js:160]: why would it? it's not a class, so what will be the difference outside?

Comment (pbazydlo) [reply src/cli/facade/dev.js:160]: We've tested it together and it does not seem to affect the initalisation after all.

**Review by pbazydlo — APPROVED** [review]

**Comment by matteofigus** [comment]
One thing about this, is making sure we update the docs like this https://github.com/opencomponents/oc/wiki#setup-a-registry

**Comment by ricardo-devis-agullo** [comment]
Technically you can still call it with `new`, but yeah, I'll remove it too.

**Pull request merged**